File: layout_1/app/src/main/java/com/example/layout_1/IntroActivity.java
package com.example.layout_1;

import android.os.Bundle;
import android.view.View;
import android.widget.ImageButton;

import androidx.appcompat.app.AppCompatActivity;

public class IntroActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_intro);

        ImageButton open = (ImageButton) findViewById(R.id.imageButton);

        open.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                CustomDialogClass dialog = new CustomDialogClass(IntroActivity.this);
                dialog.show();
            }
        });

    }
}

File: aOS_framework_demo/app/src/main/java/com/example/aos_framework_demo/IntroActivity.kt
package com.example.aos_framework_demo

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.example.aos_framework_demo.databinding.ActivityIntroBinding

class IntroActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityIntroBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.demo1Extension.setOnClickListener {
            val intent = Intent(this@IntroActivity, MainActivity::class.java)
            startActivity(intent)
        }
        binding.demo2Extension.setOnClickListener {
            val intent = Intent(this@IntroActivity, ExtensionActivity::class.java)
            startActivity(intent)
        }
        binding.demo1CustomView.setOnClickListener {
            val intent = Intent(this@IntroActivity, RecyclerActivity::class.java)
            startActivity(intent)
        }
        binding.demo2CustomView.setOnClickListener {
            val intent = Intent(this@IntroActivity, StickyActivity::class.java)
            startActivity(intent)
        }
        binding.demo3CustomView.setOnClickListener {
            val intent = Intent(this@IntroActivity, ElandActivity::class.java)
            startActivity(intent)
        }
    }
}